site/components/buttons/index.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable react/no-danger */
+// @flow
+import React from 'react';
```

Comment:
Could we have a newline after the top comments? Think that helps to separate them visually from the imports.